data/required_tasks.yml

```diff
@@ -21,7 +20,6 @@ pipeline-required-tasks:
         - [git-clone, git-clone-oci-ta]
         - init
         - inspect-image
-        - show-sbom
```

Comment:
FYI, you don't need to remove the entry for each item in the list, just the most recent one (first one).